todo/src/App.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import './App.css';
+import TodoList from './components/TodoList';
+import TodoForm from './components/TodoForm';
+import { addTodo } from "./actions";
+import { connect } from 'react-redux';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      inputValue: '',
+    }
+  }
+
+  addInputValueHandler = (event) => {
+    this.setState({
+      inputValue: event.target.value
+    })
+  }
+
+  submitFormDataHandler = (event) => {
+    const newTask = {
+      id: new Date(),
```

Comment:
This is a good way to get a unique ID; however, your dummy data in your reducer has sequential ids (e.g. 0, 1, 2, etc), but when you add new ids through the input the ids are in date format. Be consistent with the formatting.